src/main/java/com/gotocompany/depot/maxcompute/converter/payload/PayloadConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.gotocompany.depot.maxcompute.converter.payload;
+
+import com.google.protobuf.Descriptors;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public interface PayloadConverter {
```

Comment:
Awesome 👏  

Add javadoc for the interface and also write what is the expectation of canConvert and convertSingular methods.

Reply:
Let's complete all the javadoc in the upcoming phase